contrib/internal/httptrace/httptrace.go

```diff
@@ -0,0 +1,61 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+// Package httptrace provides functionalities to trace HTTP requests that are commonly required and used across
+// contrib/** integrations.
+package httptrace
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+// StartRequestSpan starts an HTTP request span with the standard list of HTTP request span tags. URL query parameters
+// are added to the URL tag when queryParams is true. Any further span start option can be added with opts.
+func StartRequestSpan(r *http.Request, service, resource string, queryParams bool, opts ...ddtrace.StartSpanOption) (tracer.Span, context.Context) {
+	path := r.URL.Path
+	if queryParams {
+		path += "?" + r.URL.RawQuery
+	}
+	opts = append([]ddtrace.StartSpanOption{
+		tracer.SpanType(ext.SpanTypeWeb),
+		tracer.ServiceName(service),
+		tracer.ResourceName(resource),
+		tracer.Tag(ext.HTTPMethod, r.Method),
+		tracer.Tag(ext.HTTPURL, path),
+		tracer.Tag(ext.HTTPUserAgent, r.UserAgent()),
+	}, opts...)
```

Comment:
Should we also add the option `tracer.Measured()` too here?
Almost every integration is setting it so I was wondering what would be the impact of having it by default?
It would remove a few more lines from the integrations by putting it in common here.

Reply:
I don't see why not! If it's ever a problem, we can always refactor because this is internal code!

Reply:
Done in https://github.com/DataDog/dd-trace-go/pull/1286/commits/fcd3cd2b64b61e8d8479db9ec4cecf60e6c76ec7